experimental/collator/src/lib.rs

```diff
@@ -743,151 +730,148 @@ mod tests {
-        }
-    }
-
-    #[test]
-    fn test_sv() {
-        // This is the same as test_fi. The purpose of this copy is to test that
-        // Swedish defaults to "reformed", which behaves like Finnish "standard",
-        // and not to "standard", which behaves like Finnash "traditional".
-
-        // Adapted from ficoll.cpp in ICU4C
-        // Testing that w and v behave as in the root collation is for checking
-        // that the sorting collation doesn't exhibit the behavior of the search
-        // collation, which (somewhat questionably) treats w and v as primary-equal.
-        let left = [
-            "wat",
-            "vat",
-            "a\u{FC}beck",
-            "L\u{E5}vi",
-            // ICU4C has a duplicate of the first case below.
-            // The duplicate is omitted here.
-            // Instead, the subsequent tests are added for ICU4X.
-            "\u{E4}",
-            "a\u{0308}",
-        ];
-        let right = [
-            "vat",
-            "way",
-            "axbeck",
-            "L\u{E4}we",
-            // ICU4C has a duplicate of the first case below.
-            // The duplicate is omitted here.
-            // Instead, the subsequent tests are added for ICU4X.
-            "o",
-            "\u{E4}",
-        ];
-        let expectations = [
-            Ordering::Greater,
-            Ordering::Less,
-            Ordering::Greater,
-            Ordering::Less,
-            Ordering::Greater,
-            Ordering::Equal,
-        ];
-        let locale: Locale = langid!("sv").into();
-        let data_provider = icu_testdata::get_provider();
-
-        let mut options = CollatorOptions::new();
-        options.set_strength(Some(Strength::Tertiary));
-
-        {
-            let collator: Collator =
-                Collator::try_new(locale.clone(), &data_provider, options).unwrap();
-            check_expectations(&collator, &left, &right, &expectations);
-        }
-
-        options.set_strength(Some(Strength::Primary));
-
-        {
-            let collator: Collator =
-                Collator::try_new(locale.clone(), &data_provider, options).unwrap();
-            check_expectations(&collator, &left, &right, &expectations);
-        }
-    }
-
-    // TODO: This test should eventually test fallback
-    // TODO: Test Swedish and Chinese also, since they have unusual
-    // variant defaults. (But are currently not part of the test data.)
-    #[test]
-    fn test_region_fallback() {
-        // There's no explicit fi-FI data.
-        let locale: Locale = "fi-u-co-standard".parse().unwrap();
-
-        // let locale: Locale = langid!("fi-FI").into();
-
-        let data_provider = icu_testdata::get_provider();
-
-        let collator: Collator =
-            Collator::try_new(locale, &data_provider, CollatorOptions::new()).unwrap();
-        assert_eq!(collator.compare("ä", "z"), Ordering::Greater);
-    }
+    // #[test]
```

Comment:
Suggestion: instead of commenting this out, add the `#[ignore]` attribute. That way this stays code that gets formatted, clippied, refactored, etc.